scripts/generate_json_specs.py

```diff
@@ -22,8 +22,8 @@ def export_json_schemas(folder: Path, spec=bioimageio.spec):
         model_format_version_wo_patch = "latest"
         general_format_version_wo_patch = "latest"
     else:
-        model_format_version_wo_patch = spec.__name__.split(".")[-1]
-        general_format_version_wo_patch = ".".join(get_args(spec.raw_nodes.GeneralFormatVersion)[-1].split(".")[:2])
+        model_format_version_wo_patch = ".".join(get_args(spec.raw_nodes.ModelFormatVersion)[-1].split(".")[:2]).replace(".", "_")
+        general_format_version_wo_patch = ".".join(get_args(spec.raw_nodes.GeneralFormatVersion)[-1].split(".")[:2]).replace(".", "_")
```

Comment:
I think you added `v` before the version number.
I have a shorter version:
```
"v" + "_".join(get_args(spec.raw_nodes.GeneralFormatVersion)[-1].split(".")[:2])
```

Please make sure the links in README work!

Reply:
would you be fine with leaving out the v everywhere? It has to be in the module names, but I think elsewhere we should rather not have it to avoid confusion

Reply:
Of course, I just wanted to fix the the links in README ;)